en_US/gateway/mqttsn.md

```diff
@@ -2,17 +2,45 @@
 
 ## Introduction
 
+<!--**Add an introductory section**: Begin the page briefly introducing the relevant protocol and the Gateway. This will provide context for users who are new to the concept.-->
+
+MQTT-SN (MQTT for Sensor Networks) is a lightweight pub/sub protocol for wireless sensor networks, the EMQX MQTT-SN Gateway allows these devices to connect and communicate with EMQX, bridging the gap between MQTT-SN and the standard MQTT protocol.
```

Comment:
"These devices" seems vague. Do they refer to devices (sensors?) in wireless sensor networks? 

There should be a full stop after "wireless sensor networks".